<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>train</string>
   </property>
  </widget>
  <widget class="QLabel" name="numPicture">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>220</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="percentTrain">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="checkNameButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>checkname</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textName">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
